src/Mutator/InterfaceParentTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © 2017 Volodimir Melko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+declare(strict_types=1);
+
+namespace Infection\Mutator;
+
+use Infection\Visitor\ParentConnectorVisitor;
+use PhpParser\Node;
+
+/**
+ * Checks if given node belongs to Interface
+ */
+trait InterfaceParentTrait
```

Comment:
why do we need trait here?

1. this code is not used anywhere else
2. maybe it should be just in mutator?

I'm not a big fan of traits, but probably you have good arguments?

Reply:
I was thinking about other potential mutators that would not apply to the interface. In this case, duplicate code will appear